src/core/udb-event.factory.js

```diff
@@ -298,7 +298,7 @@ function UdbEventFactory(EventTranslationState, UdbPlace, UdbOrganizer) {
     },
 
     /**
-     * Get the calendar for this event.
+     * Get the location for this event.
```

Comment:
This comment is redundant anyway so you could just as well remove it. The fact that it was incorrect shows that it has no added value and can only get outdated anyway.